redbeat/schedulers.py

```diff
@@ -29,6 +28,9 @@
 
 from .decoder import RedBeatJSONEncoder, RedBeatJSONDecoder, to_timestamp
 
+
+logger = get_logger('celery.beat')
```

Comment:
should we call this logger 'redbeat' rather than possibly confuse with 'celery.beat' upstream logging?

Reply:
Like I mentioned in the PR description, since this is a drop in replacement for the original Celery Beat scheduler running within Beat I think it is best to log as `celery.beat`. This way you can switch schedulers and you'll get almost exactly the same logging without additional config.

On info, upstream doesn't log anything, on debug you get an additional log when beat runs the scheduler loop. I think there is little potential for confusion.

Reply:
Makes sense now thanks.